tests/rados/test_scrub_parameters.py

```diff
@@ -0,0 +1,283 @@
+                if not status:
+                    log.error(
+                        f"After the {val} none of the pgid is set as true and the waiting time is 40 minutes"
+                    )
+                    return 1
+                forced_flag = is_forced_flag_set_false(
+                    scrub_object, selected_osd, pg_id
+                )
+                if not forced_flag:
+                    log.error(
+                        "The forced flag is not set to  True and the waiting time is 40 minutes"
+                    )
+                    return 1
+                log.info(f"===Scenario-1: Completed by {val}bing the osd===")
+                pg_dump_scrub = scrub_object.get_pg_dump_scrub(selected_osd, pg_id)
+                log.info(f"The {pg_id} dumb scrub is - {pg_dump_scrub}")
+                pg_current_sched_time = pg_dump_scrub["sched_time"]
+                # Get the dump_scrub of the PG id
+                scheduled_time = verify_scheduled_time(
+                    mon_object=mon_obj,
+                    prev_dump_scrub=dump_scrub_before_scrub,
+                    pg_id=pg_id,
+                    pg_current_sched_time=pg_current_sched_time,
+                )
+                if not scheduled_time:
+                    log.error("The scrub time is greater than upper bound time")
+                    return 1
+                log.info(f"===Scenario-2:Completed by {val}bing the osd")
+                log.info(f"=====Verification completed by {val}bing the OSD====")
+                # deleted pool.This is due to execution time is taking more time until waiting for the cluster to clean
+                # state
+                method_should_succeed(rados_object.delete_pool, pool_name)
+                time.sleep(10)
+                rados_object.log_cluster_health()
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Execution of finally block")
+        rados_object.configure_pg_autoscaler(default_mode="on")
+        time.sleep(10)
+        method_should_succeed(rados_object.delete_pool, pool_name)
+        rados_object.log_cluster_health()
+    return 0
+
+
+def create_pool_get_osd(rados_object, pool_type, pool_config):
+    """
+    Method is used -
+        1. Create replicated/Erasure pool
+        2. Push the data into pool
+        3. From the active OSD list return a random OSD
+    Args:
+        rados_object: Rados object
+        pool_type: type of pool.
+        pool_config: pool configurations
+    Returns:
+        Return primary OSD ID from an active OSD list
+    """
+    rados_object.configure_pg_autoscaler(default_mode="off")
+    time.sleep(10)
+    pool_name = pool_config["pool_name"]
+    if pool_type == "replicated_pool":
+        if not rados_object.create_pool(**pool_config):
+            log.error("Failed to create the replicated Pool")
+            return 1
```

Comment:
Return None here. 1 can be mistaken for a OSD ID.

Reply:
Code modified and returning None.